### PR TITLE
Compatible with Django 2.2 and fixes

### LINKED ISSUE #4: Django 2.2 Capability
`django_rq_wrapper/management/commands/rqworkers.py`
`from django.utils.autoreload import reloader_thread`
gives and import error

We are currently getting an error with the latest version of Django. 
In Django 2.2 the autoreloader was rewritten so we need to figure out a function to hook into here
https://github.com/istrategylabs/django-rq-wrapper/blob/0e27133f63777461de7e9d2f9bfb8cb0c48afb1b/django_rq_wrapper/management/commands/rqworkers.py#L89

### PR DESCRIPTION
**Pull request by lucas-six**
fix #4

### CODE REVIEW

File: django_rq_wrapper/management/commands/rqworkers.py [hunk 1]
```diff
@@ -76,8 +61,7 @@ def handle(self, *args, **options):
 
         pid = options.get('pid')
         if pid:
-            with open(os.path.expanduser(pid), "w") as fp:
-                fp.write(str(os.getpid()))
+            Path(pid).expanduser().write_text(str(os.getpid()))
```
Comment (joesolly) [inline django_rq_wrapper/management/commands/rqworkers.py:64]: This isn't backwards compatible with pre Python 3.4, but that is probably ok at this point

**Review by joesolly — APPROVED** [review]

**Comment by CorneliusIV** [comment]
@bashhack Do you mind looking at this when you get a minute

**Comment by lostb1t** [comment]
ping

**Comment by thomasmatecki** [comment]
👋

**Comment by karol-bujacek** [comment]
Any progress with this pull-request?

**Comment by hannonq** [comment]
I'm also interested in this pull request. Any chances it will be merged?

**Comment by jcarbaugh** [comment]
@karol-bujacek @hannonq just wanted to give you all and everyone else an update on the status of this project. The original author, @joesolly, was given admin access to the repo this past week with the intention that he become maintainer of the project. He created django-rq-wrapper while we were both employed at a company that went under last year. That company's defunct account was the sole owner on PyPI but we just got access to that this past week as well so hopefully a new release will be possible very soon. Thanks!

**Comment by hannonq** [comment]
@jcarbaugh that's great to hear. I'll keep an eye on PyPI for a new release.

**Comment by joesolly** [comment]
Version 3.1 uploaded to pypi, sorry for the delay and thanks for the contribution!

**Comment by hannonq** [comment]
Nice! Only the flag `--autoreload` doesn't seem to work. 

```
python manage.py rqworkers high default low --autoreload
2020-04-20 11:23:08 - INFO - Worker rq:worker:4d79f0bdd4ea4718975bf7a41372b872: started, version 1.3.0
2020-04-20 11:23:08 - INFO - *** Listening on high, default, low...
2020-04-20 11:23:08 - INFO - Warm shut down requested
```
Any ideas?

But starting multiple workers works again :)

**Comment by joesolly** [comment]
@hannonq Can you please start a new issue for this and include more information like your version of django and python?

**Comment by hannonq** [comment]
Sorry for the late reply. I've opened a new issue. Thanks :)

**Pull request merged**